opal/datatype/opal_convertor.h

```diff
@@ -183,10 +181,6 @@ static inline int32_t opal_convertor_need_buffers(const opal_convertor_t *pConve
 {
     if (OPAL_UNLIKELY(0 == (pConvertor->flags & CONVERTOR_HOMOGENEOUS)))
         return 1;
-#if OPAL_CUDA_SUPPORT || OPAL_ROCM_SUPPORT
```

Comment:
Should this be retained, with a change to checking the CONVERTOR_ACCELERATOR & CONVERTOR_ACCELERATOR_UNIFIED flags?

Reply:
We moved this functionality upward outside this function.